PlaceRouteHierFlow/placer/SeqPair.cpp

```diff
@@ -609,6 +609,18 @@ bool SeqPair::ChangeSelectedBlock(design& caseNL) {
   return true;
 }
 
+bool SeqPair::ValidateSelect(design & caseNL){
```

Comment:
@854768750 , this function always returns True. Why not remove completely?

Reply:
@854768750 ValidateSelect() is not being used anywhere; it can be removed.

Reply:
@854768750 , just curious, why comment out but not remove? Is there a reason to keep this around?